### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by beliaev-maksim**
* limit README.md line length to 99 chars
* add docs on relation and links to data libs

### CODE REVIEW

**Comment by beliaev-maksim** [comment]
I am 99.99% percent sure that integration fails **not** because of the markdown change :)

File: README.md [hunk 1]
```diff
@@ -2,87 +2,147 @@
...
+juju v2.x:
 
 ```shell
 juju relate postgresql-k8s application
 ```
 
+juju v3.x
+
+```shell
+juju integrtate postgresql-k8s application
```
Comment (marceloneppel) [inline README.md:122]:
```suggestion
juju integrate postgresql-k8s application
```

**Review by marceloneppel — CHANGES_REQUESTED** [review]
It seems that something failed in the process of packing the charm (but it succeeded in the other tests and in a later attempt), but I couldn't find any logs on the action run that should help us understand what happened.

**Review by marceloneppel — APPROVED** [review]

**Review by dragomirp — APPROVED** [review]

**Comment by marceloneppel** [comment]
Thanks for the contribution Maksim!

**Comment by beliaev-maksim** [comment]
@dragomirp @marceloneppel 
can you merge then pls ?

**Pull request merged**